site2/docs/getting-started-standalone.md

```diff
@@ -59,8 +58,17 @@ Directory | Contains
 `instances` | Artifacts created for [Pulsar Functions](functions-overview.md).
 `logs` | Logs created by the installation.
 
+#### Install other optional components
```

Comment:
maybe you can just remove this heading, and use heading 4 for the two optional components.

Reply:
Thanks, all your comments except this have been incorporated.